workers/unity/Packages/com.improbable.gdk.core/Systems/WorkerSystem.cs

```diff
@@ -98,12 +97,6 @@ protected override void OnCreate()
             Enabled = false;
         }
 
-        protected override void OnDestroy()
-        {
-            ConnectionHandler.Dispose();
-            base.OnDestroy();
-        }
-
         protected override void OnUpdate()
```

Comment:
random question - why should this be empty instead of just not being implemented?

Reply:
`ComponentSystem.OnUpdate()` is abstract